docs/src/algorithms/discretization.md

```diff
@@ -5,3 +5,62 @@ discretize
 DiscretizationMethod
 FIST
 ```
+# FIST  
+Examples for the FIST algorithm. 
+
+### Usage  
+  
+```@example FIST
+using Meshes
+using Plots; gr(size = (500,300))
+
+poly = PolyArea(Point2f[(0.22926679f0, 0.47329807f0), (0.45326096f0, 0.2665109f0), (0.51871038f0, 0.22148979f0), (0.3152017f0, 0.31646582f0), (0.5953296f0, 0.62900037f0), (0.5951828f0, 0.6113712f0), (0.74374446f0, 0.72971905f0), (0.29018405f0, 0.5538437f0), (0.25439468f0, 0.5678829f0), (0.22926679f0, 0.47329807f0)], [])
```

Comment:
You don't need to specify `Float32` literals, `Float64` will get converted anyway. So the `f0` part can be removed.